source/tutorials/source-connector.txt

```diff
@@ -25,27 +25,12 @@ Get Started with the MongoDB Kafka Source Connector
 
    .. step:: Configure the Source Connector
 
-      .. include:: /includes/tutorials/robwma-m1-arch.txt
-
       Create an interactive shell session on the tutorial Docker container
       downloaded for the Tutorial Setup using the following command:
 
-      .. tabs::
-
-         .. tab:: Intel Processors
-            :tabid: intel-cpu
-
-            .. code-block:: bash
-
-               docker run --rm --name SourceTutorialShell --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
-
-         .. tab:: Apple M1 Processors
-            :tabid: apple-m1-cpu
-
-            .. code-block:: bash
-
-               docker run --platform='linux/arm64' --rm --name SourceTutorialShell --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
+      .. code-block:: bash
 
+         docker exec -it mongo1 /bin/bash
 
       Create a source configuration file called ``simplesource.json`` with
       the following command:
```

Comment:
In line 292, you should indicate that you can't run the kc command until you exit the mongo shell